Sources/ArgumentParser/Usage/MessageInfo.swift

```diff
@@ -119,11 +120,7 @@ enum MessageInfo {
       case let error as LocalizedError where error.errorDescription != nil:
         self = .other(message: error.errorDescription!, exitCode: .failure)
       default:
-        if Swift.type(of: error) is NSError.Type {
```

Comment:
`NSError` does not exist in Foundation Essentials so I've removed this check and defaulted to `String(describing:)`. What are folks thoughts on this? Seems excessive to bring in all of Foundation just for this

Reply:
If we really want to preserve the behavior we can use reflection (`ArgumentParser` already depends on it heavily), but that's really overkill.